### DISCUSSION TITLE
tree view no longer obeying git status

### DISCUSSION THREAD

**Discussion by rafasaur** [atom/atom#22536] [Q&A]
I wish I had more info to give on this, but I'm really not sure what caused this to happen. Currently, nothing in the tree view for any of my repos are being colored based on anything. While I do see edited/inserted/deleted lines in the file panes, the tree view remains unchanged. Even changing `.gitignore` does not affect anything.

Below is the tree view, and although `index.js` has been edited and `config.js` is excluded in `.gitignore` (second image) they are the default gray. I expect `index.js` to be orange/yellow since it has been changed from what was uploaded to GitHub, and `config.js` should be a darker gray, since it should be ignored.
![image](https://user-images.githubusercontent.com/17931989/120716057-8117f680-c493-11eb-9ab5-a503e9fefe7a.png)

Below are the `.gitignore` contents, noting `config.js` should be a darker gray in the tree view, and the red line on the left indicates the pane view "knows" it's been edited since I last pushed an update:
![image](https://user-images.githubusercontent.com/17931989/120716407-013e5c00-c494-11eb-891b-ebe13e3f2cbf.png)

The only maybe weird thing I'm doing is keeping the files on Dropbox, but I've deleted and re-pulled the repos several times, uninstalled and reinstalled Atom, and am I still experiencing the issue.

**Comment by ThatXliner** [comment]
Try reloading Atom? On the repository? Maybe you don't have a git repository initialized?

**Reply by rafasaur** [reply]
Is that different than fetching/pulling?

**Reply by ThatXliner** [reply]
Can you re-phrase your question? Maybe add some GIF screen records?

**Reply by rafasaur** [reply]
I more directly addressed my issue, and uploaded some screenshots. Does that help?

**Reply by ThatXliner** [reply]
Ok, what does running `git status` in your project directory give you?
